posts/2021-09-24-travis-security.rst

```diff
@@ -0,0 +1,46 @@
+.. post:: 2021-09-24
+   :tags: security
+   :author: mrb
+
+.. role:: raw-html(raw)
+   :format: html
+
+
+Travis CI Security Incident
+===========================
+
+On September 9, 2021 one of our core devs discovered that artifacts building on
+Travis CI were being uploaded to our conda channel from PRs running on forked
+repositories. A quick investigation revealed that Travis CI was passing encrypted
+secrets to PR builds on forks. Further examination of our logs and artifacts indicated
+that this had been happening since about September 3, 2021. See this `CVE <https://nvd.nist.gov/vuln/detail/CVE-2021-41077>`_
```

Comment:
Minor suggestion: maybe say "and confirmed by the CVE" so it is not only our guess based on the logs?